freqtrade/strategy/interface.py

```diff
@@ -156,6 +157,10 @@ def __init__(self, config: dict) -> None:
                 if timeframe_to_minutes(informative_data.timeframe) < strategy_timeframe_minutes:
                     raise OperationalException('Informative timeframe must be equal or higher than '
                                                'strategy timeframe!')
+                if not informative_data.candle_type:
+                    informative_data = InformativeData(
+                        informative_data.asset, informative_data.timeframe, informative_data.fmt,
+                        informative_data.ffill, config['candle_type_def'])
```

Comment:
Looks like named tuple becomes a wrong type. I would turn `InformativeData` to a simple class and modify one field

Reply:
hah - the fact that we're not python 3.7 only (actually 3.8 only) - we can now use Dataclasses ... this seems to be a good pick for that :+1: